### PR TITLE
Fix license issue

### PR DESCRIPTION
**Pull request by shamrickus**
## What does this PR (Pull Request) do?

- [x] This PR fixes the license issue with `.jshintignore`

## Which Traffic Control components are affected by this PR?

- Weasel

## What is the best way to verify this PR?
Run weasel

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: .dependency_license [hunk 1]
```diff
@@ -31,6 +31,7 @@ RobotoSlab-Regular\.ttf$, Apache
 \.config(\.example)?$, Apache
 /\.bowerrc$, Apache
 /\.jshintrc$, Apache
+/\.jshintignore$, Apache
```
Comment (zrhoffman) [inline .dependency_license:34]: No need to ignore this file. `.jshintignore` can have the Apache license at the top, prefixed with hashes (e.g., [like we do for dockerfiles](https://github.com/apache/trafficcontrol/blob/40934f3213/infrastructure/cdn-in-a-box/traffic_ops/Dockerfile#L1-L16)).

**Comment by rawlinp** [comment]
restest this please

**Comment by rawlinp** [comment]
retest this please

**Comment by rawlinp** [comment]
For some reason it seems like the asf-ci doesn't run the build/tests for any of @shamrickus's PRs...I'm not sure why, but we should look into fixing that.

**Comment by dneuman64** [comment]
add to whitelist

**Comment by dneuman64** [comment]
retest this please

**Comment by dneuman64** [comment]
ok to test

**Pull request merged**